core/memory/banking.ts

```diff
@@ -79,30 +87,26 @@ export function handleBanking(offset: i32, value: i32): void {
 
     if (!Memory.isMBC5) {
       // Get the bottom 2 bits
-      ramBankBits = ramBankBits & 0x03;
+      ramBankBits &= 0x03;
     } else {
       // Get the bottom nibble
-      ramBankBits = ramBankBits & 0x0f;
+      ramBankBits &= 0x0f;
     }
 
     // Set our ram bank
     Memory.currentRamBank = ramBankBits;
     return;
-  } else if (!Memory.isMBC2 && offset <= 0x7fff) {
-    if (Memory.isMBC1) {
-      if (checkBitOnByte(0, <u8>value)) {
-        Memory.isMBC1RomModeEnabled = true;
-      } else {
-        Memory.isMBC1RomModeEnabled = false;
-      }
+  } else if (!isMBC2 && offset <= 0x7fff) {
+    if (isMBC1) {
+      Memory.isMBC1RomModeEnabled = checkBitOnByte(0, <u8>value);
     }
     // TODO: MBC3 Latch Clock Data
   }
 }
 
 // Inlined because closure compiler inlines
 export function getRomBankAddress(gameboyOffset: i32): i32 {
-  let currentRomBank: i32 = Memory.currentRomBank;
+  let currentRomBank: u32 = Memory.currentRomBank;
```

Comment:
I thought i32 was the fastest type since it is native to wasm? 🤔

Reply:
nope=) `u32` could be much faster because compiler and vm could optimize it better than signed integer types

Reply:
Rad thanks! 🚀